apps/meteor/app/livechat/server/lib/hooks.ts

```diff
@@ -109,3 +110,11 @@ export const onNewRoom = makeFunction(async (room: IOmnichannelRoom) => {
 		await sendToCRM('LivechatSessionStart', room);
 	}
 });
+
+export const afterTakeInquiry = makeFunction(
+	async (params: { inquiry: InquiryWithAgentInfo; room: IOmnichannelRoom }, _agent: { agentId: string; username: string }) => {
+		if (settings.get('Livechat_webhook_on_chat_taken')) {
+			await sendToCRM('LivechatSessionTaken', params.room);
+		}
```

Comment:
not sure if I like the old code style more 🤔  maybe we should agree in a definitive style?
```suggestion
		if (!settings.get('Livechat_webhook_on_chat_taken')) {
			return;
		}
		await sendToCRM('LivechatSessionTaken', params.room);
```